pollination-apps/app-developers/integrating-the-pollination-viewer.md

```diff
@@ -0,0 +1,131 @@
+## Pollination Viewer Usage
+
+Pollination App developers can easily integrate the same high performance 3D web viewer that we use throughout our apps in their own work. We offer both a Streamlit component and a React component that can be used by the Pollination community.
+
+The Streamlit component is [published on PyPi](https://pypi.org/project/pollination-streamlit-viewer/), and our React component is [available on npmjs](https://www.npmjs.com/package/lavender-vtkjs).
+
+The Streamlit component can be installed like this:
+```shell
+$ pip install pollination-streamlit-viewer
+```
+
+This guide will focus on the Streamlit component because that what will be most useful to developers of Pollination Apps. If you're interested in learning more about the React component, please reach out through our forums.
+
+The first resource you should know about is the sample viewer app: </br>
+https://app.pollination.cloud/pollination/apps/viewer-sample-app/?tab=live+app
+
+The code for this app is located here: </br>
+https://github.com/pollination-apps/pollination-viewer-sample/blob/main/app.py
+
+This app exposes all the configurations and features of pollination-streamlit-viewer.
```

Comment:
Should we also provide a link to a sample `vtkjs` file? I would think that makes it easier for people to get started.

I also think your sample app will be more playful if it loads a sample file when I open the page.

Reply:
How can we provide a sample vtkjs file for someone? Can that be committed in the repo or is there a better way to do that?